ibc-clients/ics07-tendermint/src/client_state/misbehaviour.rs

```diff
@@ -9,132 +9,142 @@ use tendermint_light_client_verifier::Verifier;
 
 use super::{ClientState as TmClientState, TmValidationContext};
 use crate::consensus_state::ConsensusState as TmConsensusState;
+use crate::context::TmVerifier;
+
+// verify_misbehaviour determines whether or not two conflicting headers at
+// the same height would have convinced the light client.
+pub fn verify_misbehaviour<ClientValidationContext>(
+    client_state: &TmClientState,
+    ctx: &ClientValidationContext,
+    client_id: &ClientId,
+    misbehaviour: TmMisbehaviour,
```

Comment:
a small nit: let's take the reference of `TmMisbehaviour` here since the ownership is not needed.